internal/v6provider/data_source_address_lists_test.go

```diff
@@ -59,16 +59,7 @@ func TestAccKatapultDataSourceAddressLists_minimal(t *testing.T) {
 					name,
 					altName,
 				),
-
-				// Expect 3 responses and we check the 2nd and 3rd entries.
-				// This is because the terraform-acc-test org has an
-				// existing `Public DNS Servers` address list.
 				Check: resource.ComposeAggregateTestCheckFunc(
-					resource.TestCheckResourceAttr(
-						"data.katapult_address_lists.lists",
-						"address_lists.#",
-						"3",
-					),
```

Comment:
I removed this, as absolute count will vary when running multiple acceptance tests in parallel. Instead we simply check if the specific address lists we created are included in the list.